src/tools/annotation/CircleRoiTool.js

```diff
@@ -516,8 +530,14 @@ function _calculateStats(image, element, handles, modality, pixelSpacing) {
       ((pixelSpacing && pixelSpacing.rowPixelSpacing) || 1)) /
       2);
 
+  const [roundedArea, roundedDigits] = rounding.roundValue(area, uncertainty);
+  const [roundedUncertainty, roundedUncertaintyDigits] = rounding.roundValue(
+    uncertainty,
```

Comment:
should this not be the measured value instead of the uncertainty? I think I am missing something here

Reply:
roundValue() is used twice here as both measure value and uncertainty need to be rounded. roundedDigits will be used to direct the number of digit to display.

Reply:
Got it 👍 
Then maybe we can find a more suitable name for the function, because now it feels a little strange to round value and value being an uncertainty value.
Would you consider renaming to sth like "roundISOXXX(value, uncertainty)". I am not a 100% sure this would improve the clarity though.

It seems roundedUncertaintyDigits is not used, can be remove it?